sdk/maps/Azure.Maps.Rendering/src/Models/ImagePathStyle.cs

```diff
@@ -25,7 +25,7 @@ public class ImagePathStyle
         public ImagePathStyle(IList<GeoPosition> pathPositions)
         {
             Argument.AssertNotNull(pathPositions, nameof(pathPositions));
-            if (pathPositions?.Count == 0)
+            if (pathPositions == null || pathPositions.Count == 0)
```

Comment:
@MikeAlhayek:  Can you please add tests that cover the scenario that you're attempting to fix here?

Reply:
looking more into this. I don't think this is a problem since `Argument.AssertNotNull(pathPositions, nameof(pathPositions));` already handles the null check of the parameter. 

I am not sure why are we doing another check. I think changing the code to this will be better

```
```
public ImagePathStyle(IList<GeoPosition> pathPositions)
{
    Argument.AssertNotNull(pathPositions, nameof(pathPositions));
    if (pathPositions.Count == 0)
    {
        throw new ArgumentException("pathPositions should not be empty list.");
    }

    PathPositions = pathPositions;
}
```
```